website/package.json

```diff
@@ -18,12 +18,13 @@
     "fix-mjs-bug": "find ../node_modules -name '*.mjs' | xargs rm"
   },
   "dependencies": {
-    "@deck.gl/core": "^8.0.3",
-    "@deck.gl/layers": "^8.0.3",
-    "@deck.gl/react": "^8.0.3",
-    "@luma.gl/constants": "^8.0.1",
-    "@luma.gl/core": "^8.0.1",
-    "@luma.gl/webgl": "^8.0.1",
+    "@deck.gl/core": "^7.0.2",
+    "@deck.gl/layers": "^7.0.2",
+    "@deck.gl/react": "^7.0.2",
+    "@luma.gl/addons": "^7.0.0",
+    "@luma.gl/constants": "^7.0.0",
+    "@luma.gl/core": "^7.0.0",
+    "@luma.gl/webgl": "^7.0.0",
```

Comment:
😨

Reply:
I just did this temporarily until we get it working with deck.gl 8. I'll fix the deck.gl 8 integration first before landing this.